prow/cmd/generic-autobumper/bumper/bumper.go

```diff
@@ -1004,7 +1006,19 @@ func GerritNoOpChange(changeID, hostRepo string) (bool, error) {
 
 }
 
-func createCR(msg, branch, changeID, hostRepo, cookieFile string, reviewers, cc []string, stdout, stderr io.Writer) error {
+func createCR(msg, branch, changeID, hostRepo, cookieFile, author, email string, reviewers, cc []string, stdout, stderr io.Writer) error {
+	if err := Call(stdout, stderr, gitCmd, "config", "http.cookiefile", cookieFile); err != nil {
+  		return fmt.Errorf("unable to load cookiefile: %v", err)
```

Comment:
Can we also include `stdout` as part of error message?

Reply:
No need here. stdout is already output.